ui/pages/lock.tsx

```diff
@@ -21,7 +21,6 @@ import {
   useVeNationWithdrawLock,
 } from '../lib/ve-token'
 import ActionButton, { ActionButtonProps } from '../components/ActionButton'
-import { AllowanceWarning } from '../components/AllowanceWarning'
```

Comment:
Please delete the `AllowanceWarning.tsx` component as well.

Reply:
Thanks for mentioning it,
Removed!